### DISCUSSION TITLE
For Panel Links add capability to generate URL depending on value of variables.

### DISCUSSION THREAD

**Discussion by alexeenka** [grafana/grafana#63463] [Dashboards, Panels & Visualizations]
For Panel Link I want to generate URL, depends of the value of variable.

For example:
`
If VAR==ALL then http://aaa.com?q=abc ELSE http://aaa.com/?q=abc+AND+$VAR
`

<img width="207" alt="image" src="https://user-images.githubusercontent.com/17548159/220139377-d47f2692-2040-4689-8118-c20f43f2dea8.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82822. Feel free to continue the discussion around this there. Thank you!
